src/braket/devices/local_simulator.py

```diff
@@ -314,6 +314,7 @@ def _(
         if DeviceActionType.OPENQASM not in simulator.properties.action:
             raise NotImplementedError(f"{type(simulator)} does not support OpenQASM programs")
         program = Program(source=program.to_ir(ir_type=IRType.OPENQASM))
+        kwargs["mcm"] = True
```

Comment:
Maybe a comment on why? I don't know how this kwarg means or why it needs to be supplied. If you think it makes sense to add a comment, perhaps we can also expand the comment in the requirements that points to the simulator commit?